### PR TITLE
iptables: Add runtime dependency for alternatives.

### PR DESCRIPTION
**Pull request by Sumynwa**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./LICENSES-AND-NOTICES/SPECS/data/licenses.json`, `./LICENSES-AND-NOTICES/SPECS/LICENSES-MAP.md`, `./LICENSES-AND-NOTICES/SPECS/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 
Add runtime dependency on alternatives.

###### Change Log  
- Update Requires in spec file

###### Does this affect the toolchain?  
NO

###### Associated issues  
- https://microsoft.visualstudio.com/OS/_workitems/edit/54604746

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Pipeline build id: [Buddy Build](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=662337&view=results)

### CODE REVIEW

File: SPECS/iptables/iptables.spec [hunk 1]
```diff
@@ -130,6 +132,9 @@ fi
 %{_mandir}/man3/*
 
 %changelog
```
Comment (mfrw) [inline SPECS/iptables/iptables.spec:134]: Related PR:
- https://github.com/microsoft/azurelinux/pull/10814

**Comment by eric-desrochers** [comment]
Look like this is fixing the regression for which we revert the commit via PR #10814 

In addition to this fix, should you also re-introduce this commit that no longer exist (since it was reverted) ? 
https://github.com/microsoft/azurelinux/pull/10786/commits/882622de9a9340b1f996e98cdd71ec57777df2eb

**Comment by eric-desrochers** [comment]
Also please ensure to document this PR using the following tmpl: https://github.com/microsoft/azurelinux/blob/3.0/.github/pull_request_template.md

**Comment by Sumynwa** [comment]
Create https://github.com/microsoft/azurelinux/pull/11027 to reintroduce nftables changes.

**Pull request closed without merging**